modules/hdf5_handler/bes-testsuite/hdf5_handlerTest.at

```diff
@@ -922,4 +922,5 @@ AT_BESCMD_DAP4DATA_RESPONSE_TEST_DEFAULT([vlen_dim.h5.dap_ce.bescmd])
 
 # AT_BESCMD_BESCONF_RESPONSE_TEST([h5.cf/utf8_attr.h5.ddx.bescmd], [bes.e_utf8_false.conf], [xfail])
 # AT_BESCMD_BESCONF_RESPONSE_TEST([h5.default/utf8_attr.h5.ddx.bescmd], [bes.e_utf8_false.default.conf], [xfail])
+AT_BESCMD_BESCONF_RESPONSE_TEST([h5.default/swath_wrong_dim_rp.h5.dmr.bescmd], [bes.default.conf], [xfail])
```

Comment:
Why does this fail?

Reply:
This is because this file contains a variable dimension size that mismatches the group dimension size that has the same name. So the dmr file is not generated.  The baseline file is the expected dmr file.